README.md

```diff
@@ -122,7 +122,7 @@ you can use:
 #!/usr/bin/env bash
 
 echo "{\"price\": \"750.0\"}"  # put your price here
-sleep 60
+sleep 3600
```

Comment:
this should be run in a `while true; do` loop

Reply:
The purpose of the sleep is to control the interval at which the keeper re-reads the price. In a `do..while`, it'll never update the price if the file changes.